components/popover/Popover.js

```diff
@@ -34,8 +35,9 @@ const factory = (axis, calculatePositions, Overlay) => {
       active: true,
       backdrop: 'dark',
       color: 'neutral',
+      lockScroll: true,
       offsetCorrection: 0,
-      tint: 'lightest',
+      tint: 'normal',
```

Comment:
Why did this change?

Reply:
Oh wow, I did this to try out some errors in ui when it was linked, probably shouldn't have been in here.

 :ok: by @InstaK on 29/05/2018 15:50:13